### ISSUE TITLE
basic calculation series not working - unhandled exception

### ISSUE THREAD

**Issue by blounsbury-usbr** [usbr/Pisces#176]
It seems changes have been made such that calculation series only work with cbtt and parameter. Trying to add a simple calculation series such as this:

![image](https://user-images.githubusercontent.com/1508213/51198288-19a48080-18b1-11e9-88aa-03d7dbf90e3b.png)

Produces the following unhandled exception:

![image](https://user-images.githubusercontent.com/1508213/51198340-3d67c680-18b1-11e9-900b-dc5b96c9a528.png)

**Comment by blounsbury-usbr** [comment]
Maybe an item 3 to @tjrocha's list. If site id (cbtt) is non-empty and the series name has no commas the same exception occurs. When first opening the calculation properties dialog 'default' is shown in the parameter input box. Maybe that is what used to be the default series name in what is now the siteid (cbtt) input?

**Comment by tjrocha** [comment]
Updating regex in Reclamation.TimeSeries.Parser.VariableParser Line51 to what is shown below allows the UI to recognize the input series. Still need some work to trace down why the calculation itself won't process when prompted via right-click-calculate on the UI.
```
// get multiple matches in a single pattern.
patternAllVariables = @"(?<!"")(?<alias>\b[a-zA-Z]{1}\w*\b)(?!"")" + timePattern + @"|'(?<alias>[a-zA-Z_\s0-9,]*\b)'" + timePattern;
```

**Comment by ktarbet** [comment]
Did you get this resolved?

**Comment by blounsbury-usbr** [comment]
The code edits seem to work, though not elegant (would certainty welcome a more elegant solution). We were going to do a couple tests before closing this issue, but haven't had time yet.

**Comment by ktarbet** [comment]
The changes causing this issue are related to making pisces work as a hydromet server.  Defining a Siteid and parameter have some advantages for managing data.  You may consider automatically generating a siteid(cbtt) and parameter so there can be a table name (unique identifier for a specific series).

**Comment by blounsbury-usbr** [comment]
Ah, yeah, that would be more elegant. I'll rollback my changes and go that route. Thanks.

**Comment by ktarbet** [comment]
There is also a method in TimeSeriesDatabase.GetUniqueTableName(...) that can be helpful

**Comment by blounsbury-usbr** [comment]
Yeah, that what's I tried first, and where I pulled the DateTime format from. But the function immediately returns it's argument string if the table name is not in use without appending a DateTime which I was using as the parameter if not specified. It makes sense to me that the function doesn't do anything if the table name is already unique. Up to you if it would be ok to remove the TableNameInUse(...) check then GetUniqueTableName would work.
